src/main/java/gift/config/WebConfig.java

```diff
@@ -0,0 +1,23 @@
+package gift.config;
+
+import gift.resolver.LoginMemberArgumentResolver;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+import java.util.List;
+
+@Configuration
+public class WebConfig implements WebMvcConfigurer {
+
+    private final LoginMemberArgumentResolver loginMemberArgumentResolver;
```

Comment:
의존성 주입 👍
```suggestion
    private final List<HandlerMethodArgumentResolver> resolvers resolvers;
```
이렇게 주입받는 것도 가능합니다~